src/main/app/controller/courts/PostcodesController.ts

```diff
@@ -0,0 +1,220 @@
+  getCourtAreasOfLawErrorMsg = 'A problem occurred when retrieving the court areas of law. ';
+  familyAreaOfLawErrorMsg = 'You need to enable relevant county court areas of law: Housing, Money Claims, or Bankruptcy';
+  getPostcodesErrorMsg = 'A problem occurred when retrieving the postcodes.';
+  addErrorMsg = 'A problem has occurred (your changes have not been saved). The following postcodes are invalid: ';
+  deleteErrorMsg = 'A problem has occurred when attempting to delete the following postcodes: ';
+  moveErrorMsg = 'A problem has occurred when attempting to move the following postcodes: ';
+  moveErrorDuplicatedMsg = 'The postcode is already present on the destination court: ';
+  postcodesNotValidMsg = 'The postcode provided needs to be more than one character up to the full length of a postcode: '
+  noPostcodeErrorMsg = 'Please update the required form below and try again.'
+  duplicatePostcodeMsg = 'One or more postcodes provided already exist: ';
+  noSelectedPostcodeMsg = 'Please select one or more postcodes to delete.'
+  noSelectedPostcodeOrCourtMsg = 'Please select one or more postcodes and a court before selecting the move option.'
+
+  public async get(
+    req: AuthedRequest,
+    res: Response,
+    searchValue = '',
+    postcodes: string[] = null,
+    error = '',
+    areasOfLaw: string[] = null,
+    courtTypes: string[] = null,
+    updated = false): Promise<void> {
+    const slug: string = req.params.slug as string;
+
+    const errors: Error[] = [];
+    // If we have an error from validation when adding/removing or moving postcodes,
+    // append it
+    if (error) {
+      errors.push({text: error});
+    }
+
+    if (!areasOfLaw) {
+      await req.scope.cradle.api.getCourtAreasOfLaw(slug)
+        .then((value: AreaOfLaw[]) => areasOfLaw = value.map(ct => ct.name))
+        .catch(() => errors.push({text: this.getCourtAreasOfLawErrorMsg}));
+    }
+
+    if (areasOfLaw) {
+      areasOfLaw = this.checkCountyAreasOfLaw(areasOfLaw).map(ct => ct.replace(/\s/g, '_'));
+      if(!areasOfLaw.length){
+        errors.push({text: this.familyAreaOfLawErrorMsg});
+      }
+    }
+
+    if (!courtTypes) {
+      await req.scope.cradle.api.getCourtCourtTypes(slug)
+        .then((value: CourtType[]) => courtTypes = value.map(ct => ct.name.replace(/\s/g, '_')))
+        .catch(() => errors.push({text: this.getCourtTypesErrorMsg}));
+    }
+
+    if (!postcodes)
+      await req.scope.cradle.api.getPostcodes(slug)
+        .then((value: string[]) => postcodes = value)
+        .catch(() => errors.push({text: this.getPostcodesErrorMsg}));
+
+    const courts = await req.scope.cradle.api.getCourts();
+
+    const pageData: PostcodeData = {
+      postcodes: postcodes,
+      courts: courts,
+      slug: slug,
+      errors: errors,
+      updated: updated,
+      searchValue: searchValue,
+      isEnabled: !courtTypes.length ? false :courtTypes.some(ct => ct === 'County_Court'),
```

Comment:
`courtTypes?.some(ct => ct === 'County_Court') ?? false`

Would that work here?